### ISSUE TITLE
createcomputer should be optionnal

### ISSUE THREAD

**Issue by roumano** [djjudas21/puppet-winbind#14]
As it's write on the documentation (https://forge.puppet.com/jgazeley/winbind#createcomputer) the createcomputer  should be optionnal

But it's not the case : 

```
Error: Could not retrieve catalog from remote server: Error 500 on SERVER: Server Error: Evaluation Error: Error while evaluating a Resource Statement, Class[Winbind]: expects a value for parameter 'createcomputer' at /etc/puppet...
```

I'm using the lastest version available : 0.4.1

**Comment by djjudas21** [comment]
Thanks for the report. I've pushed a commit in a feature branch. Please could you quickly test this and make sure it works in your environment, then I'll merge and tag a new release.

**Comment by roumano** [comment]
can test it totally as i have try to use in on a debian machine and package name issue ....
anyway it's look good for me.

**Comment by djjudas21** [comment]
Thanks. I've merged, and tagged 0.4.2 which is now on GitHub and Puppet Forge.
